python_scripts/01_tabular_data_exploration.py

```diff
@@ -364,5 +364,5 @@
 #   and evaluating your machine learning model;
 # * having redundant (or highly correlated) columns can be a problem for some
 #   machine learning algorithms;
```

Comment:
```suggestion
# * columns can be redundant (or highly correlated). This is not necessary a problem
#   for machine learning models but it can make it challenging to assess the
#   relative importance of the input variables in their ability to make accurate
#   predictions.    
```